src/main/java/hudson/plugins/ccm/CcmResult.java

```diff
@@ -44,8 +44,9 @@
 	 * @param defaultEncoding
 	 * @param result
 	 */
-	public CcmResult(final AbstractBuild<?, ?> build, final String defaultEncoding, final ParserResult result) {
-		super(build, defaultEncoding, result, new BuildHistory(build, CcmResultAction.class));
+	public CcmResult(final Run<?, ?> build, final String defaultEncoding, final ParserResult result) {
+		super(build, new BuildHistory(build, CcmResultAction.class, false, false), result, defaultEncoding);
+		serializeAnnotations(result.getAnnotations());
```

Comment:
This change is not related to workflow, but it was needed after upgrading to analysis-core upstream snapshot.
See [this change](https://github.com/jenkinsci/analysis-core-plugin/commit/3185a5e28a16b0088b370a2edb6fe145a80dda46) which requires a explicit call to `serializeAnnotations(result.getAnnotations())`